build/Jenkinsfile

```diff
@@ -0,0 +1,35 @@
+pipeline {
+  agent {
+      label "gcp-agent"
+  }
+  stages {
+    stage('TF init') {
```

Comment:
Add an additional stage for parity with CB steps.
Ref: https://github.com/terraform-google-modules/terraform-example-foundation/pull/100#discussion_r460332673